$NetBSD: patch-ad,v 1.3 2011/10/01 21:42:16 dholland Exp $

- use pkgsrc curses
- void main
- use const for clean build

--- fed/fed.h.orig	1995-10-07 21:44:48.000000000 +0000
+++ fed/fed.h
@@ -32,7 +32,7 @@
  * 	fed.h, 3.00, last edit-date: [Sun Jan  2 20:10:31 1994]
  */
 
-#include <ncurses.h>
+#include <curses.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
@@ -109,9 +113,9 @@ void invert ( void );
 void setchr ( char type );
 void setrow ( char type );
 void setcol ( char type );
-void main ( int argc, char *argv[] );
+int main ( int argc, char *argv[] );
 void readfont ( char *filename );
-void dis_cmd ( char *strg );
+void dis_cmd ( const char *strg );
 void clr_cmd ( void );
 void save_ch ( void );
 void move_ch ( int src, int dest );
